setup.py

```diff
@@ -68,6 +69,7 @@
         "cuda12x": [
             "cudf-cu12>=24.2",
             "dask-cudf-cu12>=24.2",
+            "cuml-cu12>=24.2",
```

Comment:
So current behavior of crossfit is to fall back to CPU for machine learning if cuML is not present .  We need to have everything end to end on GPU. 

This will be resolved once we package crossfit up. Lets discuss if we think not having a package for crossfit is a problem. I am trying to get that resolved but did not want that to block adoption here.